main.go

```diff
@@ -124,11 +124,12 @@ func main() {
 
 	if renderManifestsFor != "" {
 		reconciler := &controllers.NUMAResourcesOperatorReconciler{
-			Log:          ctrl.Log.WithName("controllers").WithName("RTE"),
-			APIManifests: apiManifests,
-			RTEManifests: rteManifests,
-			Platform:     clusterPlatform,
-			ImageSpec:    images.ResourceTopologyExporterDefaultImageSHA,
+			Log:                 ctrl.Log.WithName("controllers").WithName("RTE"),
+			APIManifests:        apiManifests,
+			RTEManifests:        rteManifests,
+			InitialRTEManifests: rteManifests.Clone(),
```

Comment:
but `RTEManifests` is going to be overwritten each time, isnt' it? So what about just
```
Log:                 ctrl.Log.WithName("controllers").WithName("RTE"),
APIManifests:        apiManifests,
InitialRTEManifests: rteManifests,
```

Reply:
We need to keep the mutated ```RTEManifests``` somewhere.
This is why we must have two copies:
one which we always mutate - ```RTEManifests```
two which we never mutate, the original manifest - ```InitialRTEManifests```

Reply:
sure, I meant only about the initialization. The member attribute should be kept of course, for the very reason you mentioned.

Reply:
So in the beginning ```RTEManifests``` is going to be empty and then we'll assign the mutated object to it.
But we are about to change the way of how ```Update()``` works anyway - it won't return the mutated object so we'll need to clone it anyway don't we?